### ISSUE TITLE
Missing gelf header files

### ISSUE THREAD

**Issue by prologic** [netdata/kernel-collector#34]
From [build logs](https://github.com/netdata/kernel-collector/pull/32/checks?check_run_id=437510191):

```
mkdir -p ./staticobjs
cc -I. -I../include -I../include/uapi -DCOMPAT_NEED_REALLOCARRAY -fno-stack-protector -I /usr/src/linux/usr/include -D_LARGEFILE64_SOURCE -D_FILE_OFFSET_BITS=64   -c bpf.c -o staticobjs/bpf.o
cc -I. -I../include -I../include/uapi -DCOMPAT_NEED_REALLOCARRAY -fno-stack-protector -I /usr/src/linux/usr/include -D_LARGEFILE64_SOURCE -D_FILE_OFFSET_BITS=64   -c btf.c -o staticobjs/btf.o
btf.c:13:10: fatal error: gelf.h: No such file or directory
   13 | #include <gelf.h>
      |          ^~~~~~~~
compilation terminated.
make[2]: *** [Makefile:102: staticobjs/btf.o] Error 1
```

**Comment by prologic** [comment]
cc @thiagoftsm

**Comment by thiagoftsm** [comment]
We are missing some `elfutils` package for this compilation.

**Comment by prologic** [comment]
Are you sure about this?

```sh
/kernel-collector # find / | grep 'asm/mmiowb.h'
/usr/src/linux-5.4.18/arch/powerpc/include/asm/mmiowb.h
/usr/src/linux-5.4.18/arch/mips/include/asm/mmiowb.h
/usr/src/linux-5.4.18/arch/riscv/include/asm/mmiowb.h
/usr/src/linux-5.4.18/arch/ia64/include/asm/mmiowb.h
/usr/src/linux-5.4.18/arch/sh/include/asm/mmiowb.h
/kernel-collector #
```

It iti in the kernel source tree?

**Comment by thiagoftsm** [comment]
No, in my environment it is inside `/usr/include`.

**Comment by prologic** [comment]
I see

**Comment by prologic** [comment]
Not clear to me where to get this library / dev package from in our Alpine 3.11 env.

**Comment by thiagoftsm** [comment]
> Not clear to me where to get this library / dev package from in our Alpine 3.11 env.

Well, I would try https://pkgs.alpinelinux.org/package/edge/main/x86/elfutils-dev , it looks like it can have what you need.

**Comment by prologic** [comment]
I _think_ I've fixed it for the glibc build env.

**Comment by prologic** [comment]
Fixed by #36
